clickhouse/client.cpp

```diff
@@ -793,6 +813,11 @@ void Client::Impl::SendQuery(const Query& query) {
                 throw UnimplementedError(std::string("Can't send open telemetry tracing context to a server, server version is too old"));
             }
         }
+        if (server_info_.revision >= DBMS_MIN_REVISION_WITH_PARALLEL_REPLICAS) {
+            WireFormat::WriteUInt64(*output_, 0);
```

Comment:
Please add comments describing what data it skipped / written as `0`

Reply:
Im sorry. It something to replication feature. I do not know what it is exactly. You can ask your colleagues